app/index.html

```diff
@@ -11,9 +11,8 @@
     <div id="app"></div>
   </body>
 
+
   <script>
-    // This automatically transpiles ES6
-    require('babel-register');
```

Comment:
`electron-compile` now handles babel and other transpilation, so we don't need to package it with the app like this